AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_tools_utils.py

```diff
@@ -90,12 +90,29 @@ def set_pane_visibility(pane_name: str, value: bool) -> None:
     azlmbr.atomtools.AtomToolsMainWindowRequestBus(bus.Broadcast, "SetDockWidgetVisible", pane_name, value)
 
 
-def select_lighting_config(asset_path: str) -> None:
-    asset_id = asset_utils.Asset.find_asset_by_path(asset_path)
-    azlmbr.atomtools.EntityPreviewViewportSettingsRequestBus(
+def load_lighting_preset_by_asset_id(asset_id: azlmbr.math.Uuid) -> bool:
+    return azlmbr.atomtools.EntityPreviewViewportSettingsRequestBus(
         azlmbr.bus.Broadcast, "LoadLightingPresetByAssetId", asset_id)
 
 
+def load_lighting_preset(asset_path: str) -> bool:
+    return azlmbr.atomtools.EntityPreviewViewportSettingsRequestBus(
+        azlmbr.bus.Broadcast, "LoadLightingPreset", asset_path)
+
+
+def get_current_lighting_preset() -> azlmbr.math.Uuid:
+    return azlmbr.atomtools.EntityPreviewViewportSettingsRequestBus(azlmbr.bus.Broadcast, "GetLightingPreset")
+
+
+def get_last_lighting_preset_asset_id() -> azlmbr.math.Uuid:
+    return azlmbr.atomtools.EntityPreviewViewportSettingsRequestBus(
+        azlmbr.bus.Broadcast, "GetLastLightingPresetAssetId")
+
+
+def get_last_lighting_preset_path() -> str:
+    return azlmbr.atomtools.EntityPreviewViewportSettingsRequestBus(azlmbr.bus.Broadcast, "GetLastLightingPresetPath")
```

Comment:
it will return the full path to an asset:
```py
] test = azlmbr.atomtools.EntityPreviewViewportSettingsRequestBus(azlmbr.bus.Broadcast, "GetLastLightingPresetPath")
] test
C:/git/o3de/Gems/Atom/Tools/MaterialEditor/Assets/MaterialEditor/LightingPresets/neutral_urban.lightingpreset.azasset
```

its not used right now but its useful for debugging, ill add a docstring and some example returns